packages/components/src/unit-control/test/index.tsx

```diff
@@ -115,6 +115,7 @@ describe( 'UnitControl', () => {
 				withoutClassName.querySelector( '.components-unit-control' )
 			).not.toHaveClass( 'hello' );
 			expect(
+				// eslint-disable-next-line testing-library/no-container
```

Comment:
There's no better way to access this wrapper element. Alternatively, we could use `.parentElement` to traverse to it, but it wouldn't be better since it would introduce a `no-node-access` violation.

Reply:
Sometimes in these scenarios I like to use `toMatchDiffSnapshot`, and I would expect to see as the only line in the snapshot the one showing the `hello` classname

Reply:
Why not! Done in 3d7d8a9e9589b1bc7c6a199b6bce06f99fca29df. There's one more line difference, but that's expected since the two unit controls have different IDs.

Reply:
Reverted. This may generate additional differences so it's not suitable for a snapshot test.

Reply:
Good to know, thanks for testing that approach!